frontend/src/app.scss

```diff
@@ -1,18 +1,45 @@
 :global(body){
-  margin: 0;
-  padding: 0;
-  box-sizing: border-box;
-  height: 100%;
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+	height: 100%;
 }
 :global(*){
-  margin: 0;
-  padding: 0;
-  box-sizing: border-box;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+	font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
+	overflow: scroll;
+	overflow-x: hidden;
+	overflow-y: overlay;
```

Comment:
the three lines starting with `overflow` are new, making scrollbar appear in front of content instead of beside